### PR TITLE
tag: generic-actionrow

### PR DESCRIPTION
**Pull request by MrMythicalYT**
Add a tag for the generic in `ActionRowBuilder`

### CODE REVIEW

**Comment by jaw0r3k** [comment]
new select menus? 🤔

**Comment by MrMythicalYT** [comment]
> new select menus? 🤔

Those aren't even added to

File: tags/tags.toml [hunk 1]
```diff
@@ -1019,3 +1019,15 @@ If you are waiting for button or select menu input from a specific message, don'
 + <Message>.createMessageComponentCollector(…)
 ```
 """
+
+[generic-actionrow]
+keywords = ["generic-actionrow", "generic-builder", "component-builder", "generic-actionrowbuilder"]
+content = """
+When using `ActionRowBuilder` in TypeScript, you need to provide the generic, specifying what the action row holds
+```js
```
Comment (almostSouji) [inline tags/tags.toml:1027]:
```suggestion
```ts
```

minor nit, but when talking about types, should use typescript

**Review by almostSouji — CHANGES_REQUESTED** [review]

File: tags/tags.toml [hunk 2]
```diff
@@ -1019,3 +1019,15 @@ If you are waiting for button or select menu input from a specific message, don'
 + <Message>.createMessageComponentCollector(…)
 ```
 """
+
+[generic-actionrow]
+keywords = ["generic-actionrow", "generic-builder", "component-builder", "generic-actionrowbuilder"]
+content = """
+When using `ActionRowBuilder` in TypeScript, you need to provide the generic, specifying what the action row holds
```
Comment (almostSouji) [inline tags/tags.toml:1026]:
```suggestion
In TypeScript the `ActionRowBuilder` class has a generic type parameter that specifies the type of component the action row holds:
```

**Pull request merged**